src/group/tag/tag.schema.ts

```diff
@@ -0,0 +1,14 @@
+import { Schema } from 'mongoose';
+
+const toLower = (label: string): string => label.toLowerCase();
+
+export default new Schema(
```

Comment:
Why the tags field is an array of objects and not an array of strings?

Reply:
It follows @ShragaUser [comment on the user subSchema](https://github.com/AlphaWolfTeam/group-service/pull/1#discussion_r548507106). We concluded that it is better going forward because it is better for validations and hooks.
Also, we could consider adding timestamps / createdBy to the tags later on.

Reply:
If it makes it harder for the client right now, we can add a query-string to the GET requests to reduce the array of objects to  an array of tag-label strings.

Reply:
Sounds good. I think that we don't need another request for the client.